libs/core/src/lib/carousel/carousel.component.ts

```diff
@@ -284,6 +284,11 @@ export class CarouselComponent implements OnInit, AfterContentInit, AfterViewIni
         this._onDestroy$.complete();
     }
 
+    /** @hidden */
+    get _showPageIndicators(): boolean {
```

Comment:
here is the getter that just returns another getter, i think we can remove it & use original one.

Reply:
I did it for better readability, but I can use original one

Reply:
@platon-rov please check again